tests/conftest.py

```diff
@@ -44,3 +47,14 @@ def coindesk(settings: Settings) -> CoinDeskApiStub:
     coindesk_mock = CoinDeskApiStub(settings.coindesk_api_url)
     with coindesk_mock() as coindesk:
         yield coindesk
+
+
+@fixture(autouse=True)
```

Comment:
Do you actually need and want to do it for each test? Ouch. That's gonna hurt performance in the longer term where there will be many more tests

Reply:
You are right about performance. But still, this gives me stateless tests. 
For now, this simplifies testing, removing this would require changing bdd-like tests to a clean the state of the system.